content/en/docs/tasks/administer-cluster/running-cloud-controller.md

```diff
@@ -53,6 +53,13 @@ Successfully running cloud-controller-manager requires some changes to your clus
   kubelet is aware that it must be initialized by the cloud controller manager
   before it is scheduled any work.
 
+  {{< note >}}
+  The `--cloud-provider` flag is used differently for `kubelet`, `kube-apiserver` and
+  `kube-controller-manager` depending on the user. If the user has an external CCMs
```

Comment:
minor grammar nit

```suggestion
  `kube-controller-manager` depending on the user. If the user has an external CCM
```

Reply:
Hi @elmiko,
Thanks for noticing, I have update the text.
PTAL!